components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationManagementServiceImpl.java

```diff
@@ -699,12 +699,16 @@ public String getServiceProviderNameByClientId(String clientId, String clientTyp
             }
         }
 
-        ApplicationDAO appDAO = ApplicationMgtSystemConfig.getInstance().getApplicationDAO();
-        name = appDAO.getServiceProviderNameByClientId(clientId, clientType, tenantDomain);
-        if (name == null) {
-            name = new FileBasedApplicationDAO().getServiceProviderNameByClientId(clientId,
-                    clientType, tenantDomain);
+        if (!StringUtils.isEmpty(clientId)) {
```

Comment:
Shall we use `StringUtils.isNotEmpty()`?

Reply:
Fixed in ff86fb7161c9506915b66c31b2a8907fbed53f2d.